llvm/tools/llvm-profdata/llvm-profdata.cpp

```diff
@@ -3045,10 +3071,10 @@ static int show_main(int argc, const char *argv[]) {
   if (ProfileKind == instr)
     return showInstrProfile(
         Filename, ShowCounts, TopNFunctions, ShowIndirectCallTargets,
-        ShowMemOPSizes, ShowDetailedSummary, DetailedSummaryCutoffs,
-        ShowAllFunctions, ShowCS, ValueCutoff, OnlyListBelow, ShowFunction,
-        TextFormat, ShowBinaryIds, ShowCovered, ShowProfileVersion,
-        ShowTemporalProfTraces, SFormat, OS);
+        ShowMemOPSizes, ShowVTables, ShowDetailedSummary,
```

Comment:
nit: The number of params here is growing. It would be nice to refactor out the params as a struct for readability (separate patch would be nice if you choose to address this).

Reply:
sounds good. I'll send out separate patch for the refactor and update back.

Reply:
I sent out https://github.com/llvm/llvm-project/pull/71328 to do the refactor. 

Given that the refactor patch would probably be large, and this PR is large itself (touches file that needs to be merged), I'll take the liberty to resolve this in a follow-up patch.